dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -5872,6 +5878,12 @@ paths:
         description: "version"
         required: true
         type: "string"
+      - name: "hardRefresh"
+        in: "query"
+        description: "refresh all versions"
+        required: false
+        type: "boolean"
+        default: true
```

Comment:
Unclear what this query parameter does for this version specific endpoint

Reply:
"refresh all versions, even if they have not changed" Better?

Reply:
like if you're refreshing all versions, why is someone passing in a versionId

Reply:
Oh I see, good catch.